__tests__/app.test.js

```diff
@@ -234,17 +234,32 @@ describe('GET /api/articles/:article:id/comments', () => {
     });
 
     describe('PATCH /api/articles/:article_id', () => {
-        test('should ',async () => {
-            const article_id=2
+        test('should patch votes with the new vote count ',async () => {
+            const article_id=1
             const newvote=1
             let response=await request(app)
             .patch(`/api/articles/${article_id}`)
             .send({vote_inc: newvote})
            
-            expect(response.body.update[0].votes).toEqual(1)
+            expect(response.body.update[0].votes).toEqual(101)
 
             
             
+        });
+        
+    });
+
+
+
+    describe('DELETE /api/comments/:comment_id', () => {
+        test('should delete comment with given comment_id ',async () => {
+            const comment_id=1
+            let response=await request(app)
+            .delete(`/api/comments/${comment_id}`)
+            .expect(204)
+           
+                       
+            
         });
```

Comment:
make sure you consider all the test cases you can with each endpoint. e.g here we need to error tests check the comment ids as we have done with the article IDs before